crates/core_arch/src/arm/neon/mod.rs

```diff
@@ -292,6 +292,25 @@ extern "C" {
     #[cfg_attr(target_arch = "arm", link_name = "llvm.ctpop.v16i8")]
     #[cfg_attr(target_arch = "aarch64", link_name = "llvm.ctpop.v16i8")]
     fn vcntq_s8_(a: int8x16_t) -> int8x16_t;
+
+    #[cfg_attr(target_arch = "arm", link_name = "llvm.ctlz.v8i8")]
+    #[cfg_attr(target_arch = "aarch64", link_name = "llvm.ctlz.v8i8")]
+    fn vclz_s8_(a: int8x8_t) -> int8x8_t;
```

Comment:
Why are these defined here instead of in the generated code?

Reply:
If you use automatic generation, the function will be defined inside functions such as vcls_s8, so vcls_u8 must call transmute(vcls_s8(a)) instead of transmute(vcls_s8_(a))

Reply:
I don't see the problem. It's fine if we have duplicate `extern` definitions.

Reply:
On the other hand, the link format of these functions(which is llvm.ctlz.v8i8 ) is different from other functions( which is llvm.arm.neon.*).

Reply:
Maybe I should modify the syntax of the code generator to make it more flexible to generate link information?

Reply:
I think this is good enough.